torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -174,6 +190,26 @@ struct TORCH_API KinetoThreadLocalState : public ProfilerThreadLocalState {
       std::make_unique<libkineto::CpuTraceBuffer>();
 };
 
+
+std::vector<std::string> inputTypes(const at::RecordFunction& fn) {
+  std::vector<std::string> types;
+  types.reserve(fn.inputs().size());
+  for (const c10::IValue& input : fn.inputs()) {
+    if (!input.isTensor()) {
+      types.emplace_back();
```

Comment:
input.tagKind() ?

Reply:
From the time being, we only need care about scalar and tensor. What kind of scenarios that do we need care about list?